src/Nervos/Serialization.h

```diff
@@ -0,0 +1,48 @@
+// Copyright © 2017-2022 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "../BinaryCoding.h"
+#include "../Data.h"
+#include "../uint256.h"
+#include <numeric>
+
+namespace TW::Nervos {
+
+class Serialization {
+public:
+    static void encodeDataArray(const std::vector<Data>& dataArray, Data& data) {
+        uint32_t dataLength = std::accumulate(dataArray.begin(), dataArray.end(), uint32_t(0),
```

Comment:
try to use clang format this line?

Reply:
Yes, all files are formatted using VS Code clang formatter.